style.css

```diff
@@ -0,0 +1,122 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  line-height: 1;
+  letter-spacing: 1.2;
+}
+
+html {
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  width: 100vw;
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.header-section {
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+  max-width: 100vw;
+  padding-bottom: 40px;
+  background-color: #fff;
+  border-bottom-left-radius: 8rem;
+  background-image: url(./images/header-img.png);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position: top 8rem right 0;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 2rem;
+}
+
+.logo {
+  width: 7.6rem;
+  height: 2rem;
+  color: #6070ff;
+  font-size: 1.8rem;
+  font-weight: 700;
+  line-height: 2rem;
+}
+
+ul {
+  display: flex;
+  gap: 3rem;
+  list-style: none;
+}
+
+li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+}
+
+.bio {
+  margin: 20rem auto;
+}
+
+.heading {
+  max-width: 100vw;
+  max-height: 70vh;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 4rem;
+  line-height: 6rem;
+  color: #344563;
+  margin: 0 1rem;
+}
```

Comment:
- [ ] Please 🙏🏽 use px instead of rem units, even if it is a valid unit px are used in the original version and it is easy to compare them. 
- [ ] Kindly use the correct `color: #172B4D;` to be similar to the Figma template. 
- [ ] The title in the original version occupies three lines, please fix it to match the Figma template.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219333322-8f3e2cd9-eb12-4756-b9af-51fc141246c8.png) | ![image](https://user-images.githubusercontent.com/58642949/219333465-3a8cae06-4be6-4c19-b635-f79a6771f43e.png) |